src/action/info.js

```diff
@@ -62,7 +62,12 @@ class InfoAction {
       this._nav.goHome();
     } else {
       this._nav.goLoaderSyncing();
-      observe(this._store, 'syncedToChain', () => this._nav.goHome());
+      observe(this._store, 'syncedToChain', () => {
+        this._notification.display({
+          msg: 'Syncing complete',
+        });
+        this._nav.goHome();
+      });
```

Comment:
Also, we should probably add an obverse of `syncedToChain` in the `action/index.js` and display the notification there since that should cover all cases, not just the initial sync.